packages/mobile/test/schemas.ts

```diff
@@ -907,6 +908,18 @@ export const v29Schema = {
   },
 }
 
+export const v30Schema = {
```

Comment:
note for the reviewer: I wasn't sure I needed to add a new schema. Happy to remove if necessary